### ISSUE TITLE
pdm-dotenv breaks `eval $(pdm venv activate)`

### ISSUE THREAD

**Issue by deronnax** [znd4/pdm-dotenv#12]
**Describe the bug**

```
 ➜ pdm self add pdm-dotenv
Installation succeeds.
➜  eval "$(pdm venv activate)"
zsh: command not found: Loading
```
Happens whether there is a .env file in the current directory or not. Happens the same way on bash.
Which is unfortunate, because since PDM has no `pdm shell` command, `eval "$(pdm venv activate)"` is recommended in PDM doc as an official replacement for `pdm shell`. I know `package_manager shell` is locked down, but my company is heavily reliant on it.
I think the issued message should be kept (I really want to know whereas PDM is loading my .env or not), but the `-q` flag does not suppress it and I think respecting that flag would be a very satisfying fix to this problem.

**To Reproduce**
```
pdm self add pdm-dotenv
eval "$(pdm venv activate)"
```

**Expected behavior**
No error, shell loading with the .env vars

**System (please complete the following information):**
- `pdm-dotenv` version: 0.3.0
- Python version: 3.10
- OS: macos Sonoma 14.1.1

**Comment by deronnax** [comment]
I was able to get the `-q` option to work with 
```diff
diff --git a/src/pdm_dotenv/__init__.py b/src/pdm_dotenv/__init__.py
index 5a69cd5..177c3ba 100644
--- a/src/pdm_dotenv/__init__.py
+++ b/src/pdm_dotenv/__init__.py
@@ -10,11 +10,12 @@ import dotenv
 from pdm.project import ConfigItem, Project
 from pdm.signals import pre_invoke
 from pdm.core import Core
+from pdm.termui import Verbosity
 
 
 def load_dotenv(project: Project, **_: Any) -> None:
     dotenv_path = project.root / project.config["dotenv.path"]
-    project.core.ui.echo(f"Loading dotenv file {dotenv_path}")
+    project.core.ui.echo(f"Loading dotenv file {dotenv_path}", verbosity=Verbosity.NORMAL)
     dotenv.load_dotenv(dotenv_path)
```

Not sure that's what you want though, I am surprized the default level for `UI.echo` is `QUIET` and not `NORMAL`. I opened  https://github.com/pdm-project/pdm/pull/2460 to fix a dosctring issue but that bring the case.

**Comment by deronnax** [comment]
fixed in #13
